Makefile

```diff
@@ -23,19 +23,18 @@ DEV ?=n
 CONTAINER_ENGINE ?=docker
 RUN_FLAGS ?=--rm
 CHECK_CHANGED_SCRIPT=$(CWD)/changed_from_main.py
-VOLUMES = -v $(CWD):/port_dev
-ifeq ($(DEV),y)
-	VOLUMES = -v $(CWD)/fv3core:/fv3core -v $(CWD)/fv3gfs-physics:/fv3gfs-physics -v $(CWD)/stencils:/stencils -v $(CWD)/dsl:/dsl -v $(CWD)/pace-util:/pace-util -v $(CWD)/test_data:/test_data -v $(CWD)/examples:/port_dev/examples -v $(CWD)/driver:/driver
-endif
```

Comment:
Why is this removed?

Reply:
Because instead the entire `pace` directory is mounted at `/pace`.